docker-compose-e2e.yml

```diff
@@ -0,0 +1,22 @@
+version: '3.2'
+
+services:
+  iqengine:
+    build:
+      context: .
+      dockerfile: ./Dockerfile
+    image: iqengine:latest
+    environment:
+      IQENGINE_METADATA_DB_CONNECTION_STRING: mongodb://${MONGO_USERNAME}:${MONGO_PASSWORD}@mongo:27017
+    depends_on:
+      - mongo
+    ports:
+      - 3000:3000
+  mongo:
+    image: mongo:4.2
+    restart: always
+    ports:
+      - 27017:27017
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: ${MONGO_USERNAME}
```

Comment:
Where do these variables come from?

Reply:
they are mentioned in the env.sample, must be set by user in the environment